docs/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "web",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.0.16",
```

And we probably should hook the version bump script up to this file, too, in the future (along with running the docusaurus command to create a new version to release to the docs).